### ISSUE TITLE
[projmgr] Add AC5 to common.schema.json

### ISSUE THREAD

**Issue by DevilTemper** [Open-CMSIS-Pack/devtools#716]
Will there be a AC5 compiler option for a csolution.yml since the AC5.5.6.7.cmake is already there?

**Comment by jkrech** [comment]
No, the AC5.5.6.7.cmake is a left-over from the early versions of cbuildgen where AC5 was supported by the cprj format. Since Arm Compiler AC5 is no longer supported, we have decided against adding it to csolution.
